setup.py

```diff
@@ -15,7 +15,6 @@
     install_requires=[
         "asgiref~=3.0",
         "autowrapt>=1.0",
-        "protobuf>=3.13.0, <5.0.0",
```

Comment:
why is that?

Reply:
i don't know why it was included in the first place, tbh - do you know why we'd need it?

Reply:
nope. I just saw many messages in the slack that discuss protobuf, so I would be cautious with these changes

Reply:
the issue with protobuf was that it was included in the package requirements and interferes with customer installation, even though it's not (to my understanding) in use...